### DISCUSSION TITLE
Postgresql issue - Server Error 500 after Update to 16.x past a few days

### DISCUSSION THREAD

**Discussion by mafi85** [amidaware/tacticalrmm#1580] [General]
Hello,

I updated my Installation past a few day, now I got Error 500 and the postgre and mongo service didn't startup.
Any Idea what I can do to solve it?

**Server Info (please complete the following information):**
 - OS: 20.04.06
 - Browser: Safari
 - RMM Version (as shown in top left of web UI): 16.0

**Installation Method:**
  - [x ] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI): unknown 
- Agent OS: [e.g. Win 10 v2004, Server 2012 R2] Windows 10 / 11

**Describe the bug**
internal Server Error 500 after Loading tacticalrmm

**Expected behavior**

**Screenshots**

**Additional context**
 systemctl --failed
  UNIT                       LOAD   ACTIVE SUB    DESCRIPTION               
● mongod.service             loaded failed failed MongoDB Database Server   
● postgresql@14-main.service loaded failed failed PostgreSQL Cluster 14-main

mongod.service - MongoDB Database Server
     Loaded: loaded (/lib/systemd/system/mongod.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Wed 2023-08-02 20:50:40 UTC; 9min ago
       Docs: https://docs.mongodb.org/manual
    Process: 836 ExecStart=/usr/bin/mongod --config /etc/mongod.conf (code=exited, status=1/FAILURE)
   Main PID: 836 (code=exited, status=1/FAILURE)

Aug 02 20:50:40 tactical mongod[836]:   Frame: {"a":"55ABC1BAE458","b":"55ABC0702000","o":"14AC458","s":"_ZN5mongo46_mon>
Aug 02 20:50:40 tactical mongod[836]:   Frame: {"a":"55ABC16FC9A3","b":"55ABC0702000","o":"FFA9A3","s":"_ZNSt17_Function>
Aug 02 20:50:40 tactical mongod[836]:   Frame: {"a":"55ABC34A69F0","b":"55ABC0702000","o":"2DA49F0","s":"_ZN5mongo11Init>
Aug 02 20:50:40 tactical mongod[836]:   Frame: {"a":"55ABC34A6D83","b":"55ABC0702000","o":"2DA4D83","s":"_ZN5mongo21runG>
Aug 02 20:50:40 tactical mongod[836]:   Frame: {"a":"55ABC16F940D","b":"55ABC0702000","o":"FF740D","s":"_ZN5mongo12_GLOB>
Aug 02 20:50:40 tactical mongod[836]:   Frame: {"a":"55ABC1668FD9","b":"55ABC0702000","o":"F66FD9","s":"main","s+":"9"}
Aug 02 20:50:40 tactical mongod[836]:   Frame: {"a":"7F0899A08083","b":"7F08999E4000","o":"24083","s":"__libc_start_main>
Aug 02 20:50:40 tactical mongod[836]:   Frame: {"a":"55ABC16F4F9E","b":"55ABC0702000","o":"FF2F9E","s":"_start","s+":"2E>
Aug 02 20:50:40 tactical systemd[1]: mongod.service: Main process exited, code=exited, status=1/FAILURE
Aug 02 20:50:40 tactical systemd[1]: mongod.service: Failed with result 'exit-code'.

~$ tail -f /rmm/api/tacticalrmm/tacticalrmm/private/log/django_debug.log
           ^^^^^^^^^^^^^^^^^^^^^
  File "/rmm/api/env/lib/python3.11/site-packages/django/db/backends/postgresql/base.py", line 275, in get_new_connection
    connection = self.Database.connect(**conn_params)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/rmm/api/env/lib/python3.11/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.OperationalError: connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?

**Comment by dinger1986** [comment]
what does your postgres log say?

**Comment by silversword411** [comment]
Did you check 
https://docs.tacticalrmm.com/troubleshooting/#error-500

Where is your server hosted?

What steps did you take between your server working, and not working?

**Comment by mafi85** [comment]
short feedback, the harddrive was filled from the backup script.
